source/guide_timetagger.rst

```diff
@@ -0,0 +1,171 @@
+.. highlight:: console
+
+.. author:: Marvin Dickhaus <https://marvindickhaus.de>
+
+.. tag:: lang-python
+.. tag:: web
+.. tag:: accounting
+.. tag:: time-tracking
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/timetagger.svg
+      :align: center
+
+##########
+Timetagger
+##########
+
+.. tag_list::
+
+Timetagger_ is a free, open source time-tracking software written in Python. It has an interactive user experience and powerful reporting.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Python <lang-python>`
+  * :manual:`domains <web-domains>`
+  * :manual:`supervisord <daemons-supervisord>`
+
+License
+=======
+
+Timetagger is released under the `GPLv3 License`_. All relevant information can be found in the LICENSE_ file in the repository of the project.
+
+Prerequisites
+=============
+
+Your website domain needs to be set up:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+To install timetagger we use :manual_anchor:`pip <lang-python.html#pip>`.
+
+.. note:: The ``pip``-command defaults to pip from python 2.7. Please use a modern version of pip instead. See the :manual:`manual <lang-python/#update-policy>` for currently supported versions. In this guide, we use python3.11.
+
+.. code-block:: console
+ :emphasize-lines: 1
+
+ [isabell@stardust ~]$ pip3.11 install --user -U timetagger
+ Collecting timetagger
+  Downloading timetagger-22.6.6-py3-none-any.whl (2.9 MB)
+  […]
+  Successfully installed MarkupSafe-2.1.1 asgineer-0.8.1 bcrypt-3.2.2 cffi-1.15.1 click-8.1.3 h11-0.13.0 itemdb-1.1.1 jinja2-3.1.2 markdown-3.3.7 pscript-0.7.7 pycparser-2.21 pyjwt-2.4.0 timetagger-22.6.6 uvicorn-0.18.2
+  […]
+ [isabell@stardust ~]$
+
+Configuration
+=============
+
+Create main file
+----------------
+
+To configure Timetagger we need to create our own main file.
+
+.. code-block:: console
+ :emphasize-lines: 1
+
+ [isabell@stardust ~]$ cp ~/.local/lib/python3.11/site-packages/timetagger/__main__.py ~/bin/mytimetagger.py
+ [isabell@stardust ~]$
+
+Next we select a random port number (e.g. 34423) and map it in the backend.
```

Comment:
I suggest to use a simple and fixed port number like 3000

Reply:
Port 4800 it is. :-)